main.tf

```diff
@@ -110,7 +122,7 @@ resource "azurerm_template_deployment" "app_service_ssl" {
 
   template_body       = "${data.template_file.ssltemplate.rendered}"
   name                = "${var.product}-${var.env}${var.deployment_target}-cert"
-  resource_group_name = "${local.resource_group_name}"
+  resource_group_name = "${azurerm_resource_group.rg.name}"
```

Comment:
please revert this change, we're in the progress of migrating applications to always passing in a resource group to the module, as we're decommissioning this module as part of moving to Kubernetes

REQ